app/models/forms/risks/communication.rb

```diff
@@ -1,14 +1,14 @@
 module Forms
   module Risks
     class Communication < Forms::Base
-      optional_details_field :hearing_speach_sight
-      optional_details_field :can_read_and_write
-
       optional_field :interpreter_required
       property :language, type: StrictString
       validates :language,
         presence: true,
         if: -> { interpreter_required == TOGGLE_YES }
+
+      optional_details_field :hearing_speach_sight
+      optional_details_field :can_read_and_write
```

Comment:
Because the order affects the error messages on top of the page. Matt wants the list of errors you read on top of the page reflect the order in which these fields are presented on the page.